Configuration/TCA/Overrides/pages.php

```diff
@@ -34,9 +34,15 @@
                 ],
             ],
             // Show only fields useful in email context
+            'palettes' => [
+                'visibilityEmailTemplate' => [
+                    'label' => 'LLL:EXT:frontend/Resources/Private/Language/locallang_tca.xlf:pages.palettes.visibility',
+                    'showitem' => 'hidden;LLL:EXT:core/Resources/Private/Language/locallang_tca.xlf:pages.hidden_toggle_formlabel',
```

Comment:
```suggestion
                    'showitem' => 'hidden',
```

If not specifically needed I would not set this label explicitly — it makes changing the label via PageTS easier.